acceptance/tests/ssl/certificate_extensions.rb

```diff
@@ -28,7 +28,7 @@
       'environmentpath' => environments_dir,
     },
     'master' => {
-      'autosign' => '/bin/true',
+      'autosign' => true,
```

Comment:
This test doesn't care how the autosign happens so we don't need to use the executable form here